scripts/post-link-android.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
```

Comment:
I think, we can remove this file because we does not use any addition android dependencies from `jitpack.io`